app.py

```diff
@@ -4,9 +4,18 @@
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
+@app.route('/', methods = ['POST', 'GET', 'PUT', 'DELETE'])
 def index():
-    return 'Hello World!'
+    if request.method == 'POST':
+        NewItem = request.form["NewItem"]
+        session.add_item(NewItem)
+        return render_template('index.html', items = session.get_items())
+    elif request.method == 'PUT':
+        SelectedItem = requestform["CompleteTask"]
+        session.save_item()
+        return render_template('index.html', items = session.get_items())
+    else:
+        return render_template('index.html', items = session.get_items())
```

Comment:
This should be split into separate functions. And it would be nicer to use meaningful URLs for the form endpoints.